datafusion/datasource-parquet/src/opener.rs

```diff
@@ -112,6 +113,7 @@ pub(super) struct ParquetOpener {
     /// Maximum size of the predicate cache, in bytes. If none, uses
     /// the arrow-rs default.
     pub max_predicate_cache_size: Option<usize>,
+    pub reverse_scan_inexact: bool,
```

Comment:
Perhaps some comments would help -- specifically on what is inexact

Reply:
Actually, after reading this more I think a more accurate name for this flag would be `reverse_row_groups` or something to make it clear it just controls which row groups are reversed

Reply:
Good point, thanks, i will address this!